From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Yash Garg <yashgarg@murena.io>
Date: Wed, 5 Oct 2022 15:39:01 +0000
Subject: [PATCH 1/3] remove screenshot button from quickstep overview actions

[a17: same one-line change; a17 dropped the theme attribute on that button]

(cherry picked from commit e7ffe68d59adc96ad82f8aa8c24d3183322203f3)
---
 quickstep/res/layout/overview_actions_container.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/quickstep/res/layout/overview_actions_container.xml b/quickstep/res/layout/overview_actions_container.xml
index ce54028..09883ce 100644
--- a/quickstep/res/layout/overview_actions_container.xml
+++ b/quickstep/res/layout/overview_actions_container.xml
@@ -35,7 +35,8 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:drawableStart="@drawable/ic_screenshot"
-            android:text="@string/action_screenshot" />
+            android:text="@string/action_screenshot"
+            android:visibility="gone" />
 
         <Button
             android:id="@+id/action_split"
-- 
2.43.0

